scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -0,0 +1,1542 @@
+from __future__ import (absolute_import)
```

Comment:
Moving forward I would like to define some sense of order for module imports. As this is new code it would be good to reduce the amount to "cleanup". The proposal is to follow IPython's  [style guide](https://raw.githubusercontent.com/wiki/ipython/ipython/template.py). For the imports this would be as it says:

```
# List all imports, sorted within each section (stdlib/third-party (including mantid)/local sans imports).
# For 'import foo', use one import per line.  For 'from foo.bar import a, b, c'
# it's OK to import multiple items, use the parenthesized syntax 'from foo
# import (a, b, ...)' if the list needs multiple lines.
# Separate stdlib, third-party, and IPython imports by a blank line.
```